### DISCUSSION TITLE
Overflow detection for uint32 on a 64-bit system

### DISCUSSION THREAD

**Discussion by hesining** [antonmedv/expr#719] [General]
I found the code change in [v1.16.8](https://github.com/expr-lang/expr/releases/tag/v1.16.8)

The check was added here:
https://github.com/expr-lang/expr/commit/45c1ae72c0b5b031cca3533886638a7d0cae3165

![20241016-1](https://github.com/user-attachments/assets/a9645323-5b3a-4110-b787-3b938e010988)

But it was removed immediately in the following commit:

https://github.com/expr-lang/expr/commit/cae6003b4797768f5a18001bdccfa47aaeaeb6a4

![20241016-2](https://github.com/user-attachments/assets/754f5ad3-2ac4-42a8-84f1-5993a0f21314)

I would like to know why it was removed and if there is any alternative way to perform the check.

**Comment by antonmedv** [comment]
I don't remember exactly, but I think something with build for 365.
